src/main/java/uk/ac/ebi/atlas/experimentpage/JsonTSnePlotController.java

```diff
@@ -48,12 +48,13 @@ public String tSnePlotWithMetadata(@PathVariable String experimentAccession,
     }
 
     @RequestMapping(value = "/json/experiments/{experimentAccession}/tsneplot/{perplexity}/expression/{geneId}",
```

Comment:
Use `@GetMapping` instead. You will remove some boilerplate.

Reply:
I use `@GetMapping` for all the methods in `JsonTSnePlotController` and `JsonMetadataSearchController`.